task/06-conditions-n-loops-tasks.js

```diff
@@ -192,8 +222,26 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
```

Comment:
Can be simplified. First define whether you need parenthesis or brackets on the right then the same thing on the left then just do the concatenation